UnitySetup/UnitySetup.psm1

```diff
@@ -103,6 +114,7 @@ class UnitySetupInstance {
         $componentTests[[UnitySetupComponent]::Vuforia]  = , [io.path]::Combine("$playbackEnginePath", "VuforiaSupport");
         $componentTests[[UnitySetupComponent]::WebGL]    = , [io.path]::Combine("$playbackEnginePath", "WebGLSupport");
 
+        Write-Verbose "path: $Path"
```

Comment:
Probably don't want this to remain in here.

Reply:
Yep, this one was for debug purposes